src/runtime/Uno.AppLoader-MonoMac/Uno.AppLoader-MonoMac.csproj

```diff
@@ -108,9 +111,9 @@
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
   </Target> -->
-  <Target Name="AfterBuild">
+  <!-- <Target Name="AfterBuild">
     <Exec Command="clang -m64 monostub.m -o bin/$(Configuration)/monostub -framework AppKit -mmacosx-version-min=10.6" />
-  </Target>
+  </Target> -->
```

Comment:
Maybe we should just delete this instead of commenting it out?

Reply:
👍